### PR TITLE
Boot crash fix matrix

### PR DESCRIPTION
**Pull request by phunkyfish**
5.5.0
- Skip initial EPG load to avoid crash scenario on boot on LibreElec/CoreElec
- New settings format and settings levels / help text

Same solution is used in pvr,vuplus. The issue is that an EPG load is generated by kodi and also the the addon. These appear to conflict and 2 channels call GetEPGForChannel at the same time. This happens occasionally some hard to capture.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,7 @@ Settings related to the EPG.
     - `LCN (Logical Channel Number) from backend` - The channel numbers as set on the backend.
     - `Channel index in backend` - Starting from 1 number the channels as per the order they appear on the backend.
 * **Reminder time (minutes before programme start)**: The amount of time in minutes prior to a programme start that a reminder should pop up.
+* **Skip intial EPG load**: Ignore the intial EPG load. Enabled by default to prevent crash issues on LibreElec/CoreElec.
```
Comment (Jalle19) [inline README.md]: Typo in initial

File: src/client.cpp [hunk 2]
```diff
@@ -882,6 +889,13 @@ PVR_ERROR GetEPGForChannel(ADDON_HANDLE handle, int iChannelUid, time_t iStart,
   if (!channelPtr)
     return PVR_ERROR_INVALID_PARAMETERS;
 
+  if (g_skippingInitialEpgLoad)
```
Comment (Jalle19) [inline src/client.cpp]: g_skip, not skipping

Comment (phunkyfish) [reply src/client.cpp]: There is a already a g_skip for the setting. This var is the state so different.

**Comment by phunkyfish** [comment]
Fixing the root cause would also be my preference. I didn’t realise this was the same issue (or at least very similar) until I came across it on this addon.

File: src/client.cpp (hunk 2 above)
Comment (Jalle19) [reply src/client.cpp]: Oh, okay

**Comment by phunkyfish** [comment]
Also add the new settings format. Ready to do if you are ok with it @Jalle19

**Comment by Jalle19** [comment]
AppVeyor build failure seems unrelated?

**Pull request merged**